libraries/scripting/type-providers/src/kotlin/script/experimental/typeProviders/generatedCode/impl/makeMethod.kt

```diff
@@ -0,0 +1,318 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
```

Comment:
The approach here looks strange to me, especially in the lack of the usages to chack the intentions. Let's discuss it online.